akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -464,6 +464,28 @@ rebalanced to other nodes.
 See @ref:[How To Startup when Cluster Size Reached](cluster.md#how-to-startup-when-a-cluster-size-is-reached)
 for more information about `min-nr-of-members`.
 
+## Health check
+
+An [Akka Management compatible health check](https://doc.akka.io/docs/akka-management/current/healthchecks.html) is included that returns healthy once the local shard region
+has registered with the coordinator. This health check should be used in cases where you don't want to receive production traffic until the local shard region is ready to retrieve locations
+for shards. For shard regions that aren't critical and therefore should not block this node becoming ready do not include them.
+
+The health check does not fail after initial success as that would result in all nodes failing their health check if the coordinator is unavailable.
```

Comment:
Maybe this sentence is giving too much internal context. If I'm not familiar with the code, I would ask myself. "What that meansk if the coordinator is unavailable? I thought this was about the shard region being ready for use"

To get the full picture, I need to understand that the ShardRegion has a reference to the coordinator and that the reference can be `None` in some situations even after successful registration. 

Instead, we could say:

```suggestion
The health check does not fail after an initial successful check. Once a shard region is registered and is operational it stays available for incoming message. 
```

Maybe last sentence is not even necessary.

Reply:
i'll go with your suggestion. I think it useful for the user to know that this health check remains healthy, we could have done it the other way. 

I don't think it is about knowing whether there is a reference to a coordinator / internals. It is whether the local node is registered, nodes can become unregistered but we've decided not to fail the health check in that case